internal/schemautil/userconfig/apiconvert/toapi.go

```diff
@@ -270,6 +270,57 @@ func itemToAPI(
 	return res, o, nil
 }
 
+// processManyToOneKeys is a function that processes many to one keys.
```

Comment:
Please add another sentence to the comments describing how we are processing this. If I gather it correctly, we set only one value for a key for ip_filters and namespaces regardless of where it was _string or _object.

Reply:
added more comments